### PR TITLE
[Web] Bereket.adminmanagepage

### PR DESCRIPTION
**Pull request by BekyH**

### CODE REVIEW

File: Front-end/glaucoma-diagnosis-system/src/App.vue [hunk 1]
```diff
@@ -1,5 +1,5 @@
 <template>
-<div>
+<div class="bg-cover min-h-screen" style="background-image: url('https://images.unsplash.com/photo-1519681393784-d120267933ba?ixid=MnwxMjA3fDB8MHxwaG90by1wYWdlfHx8fGVufDB8fHx8&ixlib=rb-1.2.1&auto=format&fit=crop&w=1124&q=100');'">
```
Comment (ZBeimnet) [inline Front-end/glaucoma-diagnosis-system/src/App.vue:2]: I don't like the background image. I think the previous one with no bg image is better, wdyt?

File: Front-end/glaucoma-diagnosis-system/src/router/index.ts [hunk 2]
```diff
@@ -21,12 +22,18 @@ const routes = [
         component: Register,
     },
     {
+
         path: '/reception',
         name: 'Reception',
         component: Reception,
+    },
+    {
+        path:'/adminmanage',
```
Comment (ZBeimnet) [inline Front-end/glaucoma-diagnosis-system/src/router/index.ts:31]: It is better if we add "/admin" prefix for the path of all admin pages. In this case, "/admin/manage"

File: Front-end/glaucoma-diagnosis-system/src/views/adminManage.vue [hunk 3]
```diff
@@ -0,0 +1,117 @@
+<template>
+        <div>
+            <div class="flex mt-3 ml-3">
+                <ul class="flex border-b">
+  <li class="-mb-px mr-1">
```
Comment (ZBeimnet) [inline Front-end/glaucoma-diagnosis-system/src/views/adminManage.vue:5]: Please do Right click -> Format :), till we setup eslint

**Review by ZBeimnet — CHANGES_REQUESTED** [review]

File: Front-end/glaucoma-diagnosis-system/src/views/adminManage.vue (hunk 3 above)
Comment (BekyH) [reply Front-end/glaucoma-diagnosis-system/src/views/adminManage.vue]: comments addressed

**Review by Tadele01 — APPROVED** [review]

**Review by ZBeimnet — APPROVED** [review]

**Pull request merged**